core-v1/src/main/scala/engine/StandardEngine.scala

```diff
@@ -260,5 +264,16 @@ object StandardEngine {
       debugging(expr).invokeDebugger(stateFromInput((_, value, window), finalState.output))
       ExprResult.WithinWindow(expr, finalState, valueResult, windowResult)
     }
+
+    override def visitZipToShortestHList[I, F[+_], WL <: HList, UL <: HList](
+      expr: Expr.ZipToShortestHList[I, F, WL, UL, OP],
+    )(implicit
+      zip: ZipToShortest.Aux[F, WL, OP, UL],
+      opO: OP[F[UL]],
+    ): PO <:< I => ExprResult[PO, I, F[UL], OP] = { implicit evPOisI =>
+      // TODO: Figure out how to create an Arrow for this type? Maybe a simpler visitor? Maybe I can rewrite this visitor?
+      val output = zip.zipToShortestWith(expr.exprHList, this)(???)(implicitly)
+      output
+    }
```

Comment:
I am skipping on implementing this because I don't plan on using the standard interpreter for the first milestone release.